Formula/lua.rb

```diff
@@ -109,82 +83,72 @@ def install
   end
 
   def pc_file; <<~EOS
-    V= 5.2
-    R= 5.2.4
-    prefix=#{opt_prefix}
+    V= 5.3
+    R= #{version}
+    prefix=#{HOMEBREW_PREFIX}
     INSTALL_BIN= ${prefix}/bin
     INSTALL_INC= ${prefix}/include
     INSTALL_LIB= ${prefix}/lib
     INSTALL_MAN= ${prefix}/share/man/man1
-    INSTALL_LMOD= #{HOMEBREW_PREFIX}/share/lua/${V}
-    INSTALL_CMOD= #{HOMEBREW_PREFIX}/lib/lua/${V}
+    INSTALL_LMOD= ${prefix}/share/lua/${V}
+    INSTALL_CMOD= ${prefix}/lib/lua/${V}
     exec_prefix=${prefix}
     libdir=${exec_prefix}/lib
     includedir=${prefix}/include
 
     Name: Lua
     Description: An Extensible Extension Language
-    Version: 5.2.4
+    Version: #{version}
     Requires:
-    Libs: -L${libdir} -llua -lm
+    Libs: -L${libdir} -llua.5.3 -lm
     Cflags: -I${includedir}
     EOS
   end
 
-  def caveats; <<~EOS
-    Please be aware due to the way Luarocks is designed any binaries installed
-    via Luarocks-5.2 AND 5.1 will overwrite each other in #{HOMEBREW_PREFIX}/bin.
-
-    This is, for now, unavoidable. If this is troublesome for you, you can build
-    rocks with the `--tree=` command to a special, non-conflicting location and
-    then add that to your `$PATH`.
-    EOS
-  end
-
   test do
-    system "#{bin}/lua", "-e", "print ('Ducks are cool')"
+    system "#{bin}/lua-5.3", "-e", "print ('Ducks are cool')"
```

Comment:
IMO, keep this as `lua` since that's what the actual executable is called. We just install the symlink for compatibility mostly.